### PR TITLE
cart and wishlist functionality

### PR DESCRIPTION
**Pull request by SumitSaundadkar**

### CODE REVIEW

**Review by iHarryD — APPROVED** [review]
- Would recommend destructuring props.
- Use some extension to improve readability and for proper code alignment.

**Review by TusharJade — APPROVED** [review]
functionality is working awesome just see what harry is saying I will also suggest same other than that evrything looks great good to go for merge

File: src/App.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { StorePage } from "./Componets/StorePage/store";
 import { MyCartPage } from "./Pages/AddToCardCart/cart";
 import {WishlistPage} from './Pages/wishlispage/wishList'
 import { BrowserRouter, Routes, Route,Link} from "react-router-dom";
+import {LoginPage} from './Pages/LoginPage/Login';
```
Comment (SinchanaMS) [inline src/App.js:8]: Keep all the pages in one index file and import it from there to keep the code concise.

File: src/Componets/MainCrad/ProductCard.js [hunk 2]
```diff
@@ -1,24 +1,21 @@
+import "./product.css";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { useFilterContext } from "../../Contexts/filter-context";
+import { useCart } from "../../Contexts/cartContext";
+import { Link } from "react-router-dom";
+import { useWishlistContext } from "../../Contexts/wishListContext";
```
Comment (SinchanaMS) [inline src/Componets/MainCrad/ProductCard.js:7]: Keep the contexts in one index file and import them from there to reduce the lines of code.

File: src/Componets/cards/singlecard.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.content-page {
+    text-align: center;
+    display: inline-flex;
+    margin: 20px;
```
Comment (SinchanaMS) [inline src/Componets/cards/singlecard.css]: use rem units for more than 16px.

File: src/Pages/wishlispage/wishList.js [hunk 4]
```diff
@@ -1,24 +1,110 @@
...
+                            })
+                          }
+                        >
+                          <i className="fas fa-shopping-cart"></i> Add to cart
+                        </button>
+                      </div>
+                      
+                    )}
+                    
+
```
Comment (SinchanaMS) [inline src/Pages/wishlispage/wishList.js]: Remove unwanted spacing to enhance readability.

**Review by SinchanaMS — APPROVED** [review]
Suggested a few changes. 
Please keep even spacing between the lines and remove unwanted ones.

File: src/Pages/wishlispage/wishList.js (hunk 4 above)
Comment (SumitSaundadkar) [reply src/Pages/wishlispage/wishList.js]: done.

**Pull request merged**